hs_core/tests/api/native/test_irods_user_zone_federation.py

```diff
@@ -109,7 +109,20 @@ def test_resource_operations_in_user_zone(self):
         self.assertEqual(res.resource_federation_path, fed_path)
         # test original file in user test zone is removed after resource creation
         # since True is used for move when creating the resource
-        self.assertFalse(self.irods_storage.exists(user_path + self.file_to_be_deleted))

```

Comment:
This is a comment for line 92 because github won't let me comment on a line that hasn't changed.  It feels wrong to skip a unit test if the environment isn't setup correctly so I think in that case it should at least log out a warning.

Reply:
@sblack-usu Understood on your concern. The legacy for this was that we added iRODS federation-related tests after we have local iRODS containers in our local development environment, but the local iRODS container set up was not hooked up with Jenkins initially for some time during which period we only run these federation-related tests nightly on dev VM. Since these federation-related tests were not expected to run, we did not treat it as an error or warning, but with the current setup, agreed it is better to add a log there just in case there was an issue with local iRODS container setup. I'll add logging a warning message there.

Reply:
I like that even better 👍